Packs/Orca/Integrations/Orca/README.md

```diff
@@ -4,19 +4,21 @@ This integration was integrated and tested with Orca
 
 1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
 2. Search for Orca.
-3. Click **Add instance** to create and configure a new integration instance.
+ Click **Add instance** to create and configure a new integration instance.
 
     | **Parameter** | **Description** | **Required** |
     | --- | --- | --- |
     | apikey | API Key | True |
+    | api_host | API Host without schema. Default: `api.orcasecurity.io` | False
     | first_fetch | First fetch timestamp \(`<number>` `<time unit>`, e.g., 12 hours, 7 days\) | False |
     | incidentType | Incident type | False |
     | isFetch | Fetch incidents | False |
     | max_fetch | Max fetch | False |
+    | fetch_limit | Limit number of fetched records. Default: `1000` | False |
```

Comment:
I don't see this parameter in the orca.yml file

Reply:
Fixed